src/backend/joanie/core/models/courses.py

```diff
@@ -164,7 +164,57 @@ class Organization(parler_models.TranslatableModel, BaseModel):
     )
     signature = models.ImageField(_("signature"), blank=True)
     logo = ThumbnailerImageField(_("logo"), blank=True)
-    country = CountryField(_("country"), default=settings.JOANIE_DEFAULT_COUNTRY_CODE)
+    country = CountryField(
+        _("country"),
+        help_text=_(
+            "Country field will be deprecated soon in order to be replaced by address relation."
+        ),
+        default=settings.JOANIE_DEFAULT_COUNTRY_CODE,
+    )
+    enterprise_code = models.CharField(
+        verbose_name=_("Enterprise code"),  # e.g : SIRET in France
+        max_length=50,
+        blank=True,
+        null=True,
+    )
+    activity_category_code = models.CharField(
+        verbose_name=_("Activity category code"),  # e.g : APE in France
+        max_length=50,
+        blank=True,
+    )
+    representative_profession = models.CharField(
+        verbose_name=_("Profession of the organization's representative"),
+        help_text=_("representative profession"),
+        max_length=100,
+        blank=True,
+    )
+    # This field is optional 'signatory_representative'. If it is set, you must set the
+    # field 'signatory_representative_profession'
+    signatory_representative = models.CharField(
+        verbose_name=_("Signatory representative"),
+        help_text=_("signatory representative"),
+        max_length=100,
+        blank=True,
+    )
+    signatory_representative_profession = models.CharField(
+        verbose_name=_("Profession of the signatory representative"),
+        help_text=_("signatory representative profession"),
+        max_length=100,
+        blank=True,
+    )
```

Comment:
As [the related issue](https://github.com/openfun/joanie/issues/589) to this PR mention, those fields can be null but if the `signatory_representative` field has a value, the `signatory_representative_profession` should also have one.

So I think a db constraint is missing to ensure that.

Reply:
I've added a constraints on the model Organization where I want both fields set if or another is nourished with a value.
I've added some tests showing the constraint error.